serverThread.h

```diff
@@ -7,7 +7,7 @@ class ServerThread : public QThread
     Q_OBJECT
     void run() Q_DECL_OVERRIDE;
 public:
-    ServerThread(int _port, QObject *parent = 0);
+    ServerThread(int _port, QObject *parent = 0, bool useClientCount = false);
```

Comment:
The `useClientCount` option is meant to be backwards compatible, so it defaults to false.